src/Evolution/Executables/NewtonianEuler/VolumeTermsInstantiation.cpp

```diff
@@ -0,0 +1,92 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <boost/preprocessor/control/if.hpp>
+
+#include "DataStructures/DataBox/PrefixHelpers.hpp"
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "Evolution/DiscontinuousGalerkin/Actions/VolumeTermsImpl.tpp"
+#include "Evolution/Systems/NewtonianEuler/System.hpp"
+#include "Evolution/Systems/NewtonianEuler/TimeDerivativeTerms.hpp"
+#include "PointwiseFunctions/AnalyticData/NewtonianEuler/KhInstability.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/NewtonianEuler/IsentropicVortex.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/NewtonianEuler/LaneEmdenStar.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/NewtonianEuler/RiemannProblem.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/NewtonianEuler/SmoothFlow.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+
+namespace {
+// The system is currently templated on the equation of state, but it's only
+// used in code that will be removed and that we don't care about. Even the
+// dependence on the thermodynamic dimension can probably be removed. In either
+// case, we don't care about the thermodynamic dimension for the explicit
+// instantiation either.
```

Comment:
The evolution systems are sometimes templated on the EOS because the char speed compute tag needs them. This makes explicit instantiations need some form of EOS, even though the parts the volume_terms instantiation cares about doesn't depend on the EOS. It probably will depend on the thermodynamic dim, i.e. does the pressure depend on just density or density and internal energy. So to answer your questions:
- evolution system
- compute_char_speed
- the evolution system might not even need to depend on the thermodynamic dim. This isn't clear yet, but I think we can template the `compute_time_derivative_terms::apply` on the thermodynamic dim instead. We probably need to decide how to do prim recovery to answer this question for GRMHD. As in, do we want runtime choosing of the recovery scheme and the EOS

Reply:
Okay, thanks for clarifying. I'd be okay leaving the comment as-is.